components/Shared/WYSYWYGComponent.tsx

```diff
@@ -0,0 +1,10 @@
+interface WYSYWYGProps {
```

Comment:
Again, it's WYSIWYG (**W**hat **y**ou **s**ee is **w**hat **y**ou **g**et) not WYSYWYG.